README.md

```diff
@@ -4,7 +4,7 @@ This tool is used to validate terraform plans before they are applied. Validatio
 
 **Note**: Using Terraform Validator does _not_ require an active installation of Forseti. Terraform Validator is a self-contained binary.
 
-Note: this tool supports Terraform v0.12 by default. To switch to use Terraform v0.11, please see the section [Terraform v0.11](#terraform-v011).
+Note: this tool supports Terraform v0.12+.
```

Comment:
Does it work with 0.13 and 0.14 currently? Or do those require extra work

Reply:
TF version support basically has to do with the format of the terraform plan JSON format, which hasn't changed significantly since 0.12, as I understand it.

Reply:
I've been testing tf validator locally with tf 13.5 without any issues.

Reply:
Confirmed to work with terraform 0.14.2 as well (same JSON `format_version`).